danger_python/__init__.py

```diff
@@ -1 +1,3 @@
 from danger_python.danger import Danger, fail, markdown, message, warn
+
+__all__ = ["Danger", "fail", "markdown", "message", "warn"]
```

Comment:
What does this do?

Reply:
This lists public symbols exported by the module (it is what you import when you use `from danger_python import *`). 

Without appending public modules to all,  [mypy](https://mypy.readthedocs.io/en/stable/) complains about unresolved symbols.